client/src/routes/search.tsx

```
@@ -0,0 +1,48 @@
+  return (
+    <main className="space-y-4">
+      <ExperienceFilters
+        onFilterChange={(filters) => {
+          navigate({
+            search: filters,
+          });
+        }}
+      />
+
+      <InfiniteScroll
+        onLoadMore={!!search.q ? experiencesQuery.fetchNextPage : undefined}
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)